econml/inference/_inference.py

```diff
@@ -559,14 +592,14 @@ def __init__(self, d_t, d_y, pred, inf_type, fname_transformer=None,
         # For effect summaries, d_t is None, but the result arrays behave as if d_t=1
         self._d_t = d_t or 1
         self.d_y = d_y
-        self.pred = pred
+        self.pred = copy.deepcopy(pred)
         self.inf_type = inf_type
         self.fname_transformer = fname_transformer
         self.feature_names = feature_names
         self.output_names = output_names
         self.treatment_names = treatment_names
 
-    @property
+    @ property
```

Comment:
I think there was a syntax error in your file at some point that led to an incorrect parse that caused the autoformatter to insert spaces in these decorators - please revert them.
```suggestion
    @property
```

Reply:
Sure, will do all!